packages/react-jsx-highcharts/src/index.js

```diff
@@ -69,6 +69,7 @@ export const WindBarbSeries = withSeriesType('WindBarb');
 export const XRangeSeries = withSeriesType('XRange');
 
 // Providers
+export { Consumer, Provider } from './components/HighchartsContext';
```

Comment:
This seems like a sensible idea, I'm just wondering if perhaps `Consumer` and `Provider` are a bit too generic.

```suggestion
export { Consumer as HighchartsConsumer, Provider as HighchartsProvider } from './components/HighchartsContext';
```

Reply:
We also use context for Charts, Axes and Series - does the context for those need exposing too?